core/src/rpc.rs

```diff
@@ -80,7 +80,20 @@ impl JsonRpcRequestProcessor {
     }
 
     pub fn get_signature_status(&self, signature: Signature) -> Option<bank::Result<()>> {
-        self.bank().get_signature_status(&signature)
+        self.get_signature_confirmation_status(signature)
+            .map(|x| x.1)
+    }
+
+    pub fn get_signature_confirmations(&self, signature: Signature) -> Option<usize> {
+        self.get_signature_confirmation_status(signature)
+            .map(|x| x.0)
+    }
+
+    pub fn get_signature_confirmation_status(
```

Comment:
I'm confused by the return value vs. name here?   Is `get_signature_confirmation_status()` returning the number of blocks since the signature was confirmed?